.buildkite/pipeline.yaml

```
@@ -0,0 +1,15 @@
+steps:
+  - label: "Install components; check formatting; build; build the ledger, run the linter, start a Parity Ethereum node and run tests."
+    commands:
+      - 'cargo fmt --all -v -- --check'
+      - 'BUILD_DUMMY_WASM_BINARY=1 RUSTFLAGS="-D warnings" cargo build --jobs=1 --all'
```

```suggestion
      - 'BUILD_DUMMY_WASM_BINARY=1 cargo build --all'
```
Probably best to move this after `clippy`, too.

---

I see - it is preferable to fail first with linting than with building. Ok, will fix.